### ISSUE TITLE
Document key bits of `SimpleSmt`

### ISSUE THREAD

**Issue by vlopes11** [0xPolygonMiden/crypto#65]
Currently, the key bits are documented as `63-bit`

https://github.com/0xPolygonMiden/crypto/blob/398af59045efe16181140475742c7535618a9c75/src/merkle/simple_smt/mod.rs#L9-L13

However, we take `u64` as arguments. This difference might create a confusion downstream.

https://github.com/0xPolygonMiden/crypto/blob/398af59045efe16181140475742c7535618a9c75/src/merkle/simple_smt/mod.rs#L153-L154

One option is to, instead, take a `Felt` as argument. But then, we also need to rework the rpo module documentation that also mentions `64-bit` - and it is important we keep consistency between these, as RPO is the backend of `SimpleSmt`

https://github.com/0xPolygonMiden/crypto/blob/398af59045efe16181140475742c7535618a9c75/src/hash/rpo/mod.rs#L60-L65

_Originally posted by @grjte in https://github.com/0xPolygonMiden/crypto/pull/64#discussion_r1109714741_

**Comment by bobbinth** [comment]
Closing as no longer relevant after #245.
